internal/animation/runner.go

```diff
@@ -12,6 +12,7 @@ type Runner struct {
 	animationMutex    sync.RWMutex
 	animations        []*anim
 	pendingAnimations []*anim
+	nextAnimations    []*anim // used by runner goroutine only
```

Comment:
This seems a bit strange naming wise. Pending and next seem like two synonyms for the same thing?

Reply:
`pendingAnimations` is the animations that have been `Start`ed during a single tick and haven't been picked up by the runner goroutine yet.

`nextAnimations` is basically "the pre-allocated slice that will be used by the runner goroutine on the next tick to become the new `animations` slice. Yeah the naming is a bit awkward, if you have a better suggestions I'm open to any!

Reply:
It can also be thought of as "the slice for accumulating the animations that will run in the next frame"

Reply:
Hmm. Yeah, I don't quite know how to name it but I think we need something better than what is there currently. Anyone reading the code for the first time will likely be confused.

Reply:
How about a descriptive comment?

Reply:
@Jacalz I added descriptive comments to all the slice declarations - hopefully that should make it understandable :)